api/entity/Genre.ts

```diff
@@ -0,0 +1,24 @@
+import { Column, Entity, ManyToMany, PrimaryGeneratedColumn } from 'typeorm';
+import { Band } from './Band';
+import { Musician } from './Musician';
+import { Event } from './Event';
+
+@Entity()
+export class Genre {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @Column('varchar', { unique: true })
+  name: string;
+
+  @ManyToMany(() => Musician, (musician) => musician.genres, {
+    onDelete: 'CASCADE',
+  })
+  musicians: Musician[];
+
+  @ManyToMany(() => Band, (band) => band.genres, { onDelete: 'CASCADE' })
+  groups: Band[];
+
+  @ManyToMany(() => Event, (event) => event.genres, { onDelete: 'CASCADE' })
+  events: Event[];
+}
```

Comment:
I don't think this belongs here. If we want to get infos on which bands have specific group or events we should do nested routes. We'll also have problems down the line with cascade delete. WDYT?

Reply:
You're right ! I'm going to change that

Reply:
Done